README.md

```diff
@@ -42,8 +42,8 @@ fn add(x: u32, y: u32) -> u64 {
 
 #[test]
 fn test_add() {
-    rdescribe("add", |ctx| {
-        ctx.describe("0 <= x + y <= u32::MAX", |ctx| {
+    rdescribe("add", (), |ctx, _| {
+        ctx.describe("0 <= x + y <= u32::MAX", (), |ctx, _| {
```

Comment:
- README example needs update
- copy this example in a file in `examples/`
- document this in the readme like "test this with `cargo run --example....`